docs/source/executing-operations/subscription-support.mdx

```diff
@@ -115,6 +116,16 @@ flowchart LR;
 
     - You can _skip_ modifying subgraph schemas that don't define any `Subscription` fields.
 
+1. [Update your Apollo Server instances to version 4 or later](/apollo-server/migration).
```

Comment:
I think it's better to use conditional here, because it's not mandatory, it's required only if your subgraphs are using Apollo Server. So I would suggest to start with something like `If you're using Apollo Server....` . I just want to not have a confusing documentation to let our users think they MUST use apollo server to make federated subscriptions work. What do you think ?

Reply:
Good suggestion